os/net/routing/routing.h

```diff
@@ -171,9 +171,7 @@ struct routing_driver {
   /**
    * Called by uIP to notify addition/removal of IPv6 neighbor entries
    *
-   * \param addr The link-layer addrress of the packet destination
-   * \param status The transmission status (see os/net/mac/mac.h)
-   * \param numtx The total number of transmission attempts
+   * \param nbr The neighbor entry
```

Comment:
Line 93 in this file is also faulty, is this code not part of the doxygen build?

Reply:
Unfortunately I am not familiar with Doxygen - it should react to such mismatches?
In any case, I fixed line 93 as well now.

Reply:
The only thing I know about Doxygen is that it sometimes does, but it sometimes doesn't. And the next revision (1.9.2->1.9.3) might warn for completely different things. When the next revision doesn't break the support for preprocessor defines, the warnings one gets tend to be valid though.

Clang has a warning that also helps (`-Wdocumentation` or similar?).

All that is outside the scope of this patch though, thanks for fixing the extra issue!